### PR TITLE
fix: Add bg colour to favicon `ImgInput` [skip pizza]

### PR DESCRIPTION
**Pull request by DafyddLlyr**
Quick fix I noticed when watching @ianjon3s video!

### CODE REVIEW

**Comment by ianjon3s** [comment]
I'm not sure about this one, the background of the browser tab will depend on the browser in use (and will most commonly be a light shade of grey for most browsers used without customisation), so this could potentially be misleading. An ideal favicon would be set up to work against either dark or light.

File: editor.planx.uk/src/pages/FlowEditor/components/Settings/DesignSettings/FaviconForm.tsx [hunk 1]
```diff
@@ -62,6 +62,7 @@ export const FaviconForm: React.FC<FormProps> = ({
           <InputRowItem width={formik.values.favicon ? 90 : 50}>
             <ImgInput
               img={formik.values.favicon || undefined}
+              backgroundColor={formik.values.primaryColour}
```
Comment (Mike-Heneghan) [inline editor.planx.uk/src/pages/FlowEditor/components/Settings/DesignSettings/FaviconForm.tsx:65]: _nit:_

I'm being super pedantic here but am I right in saying the favicon won't actually be set in a place where the background will be in effect? As in it'll be on a background which they've set their browser to?

Comment (Mike-Heneghan) [reply editor.planx.uk/src/pages/FlowEditor/components/Settings/DesignSettings/FaviconForm.tsx:65]: I'm just wondering whether the `primaryColour` might still not have enough contrast like we saw in the video with the OSL favicon?

**Comment by DafyddLlyr** [comment]
Thanks for the feedback - all valid points. Leaving as-is because this would be most accurate for the user.

**Pull request closed without merging**